apps/server/lib/lexical/server/code_intelligence/completion.ex

```diff
@@ -61,6 +61,14 @@ defmodule Lexical.Server.CodeIntelligence.Completion do
       Enum.empty?(prefix_tokens) or not context_will_give_meaningful_completions?(env) ->
         Completion.List.new(items: [], is_incomplete: true)
 
+      Env.in_context?(env, :struct_arguments) and not Env.in_context?(env, :value) and
+          not prefix_is_trigger?(env) ->
+        aliases = Env.prefix_aliases(env)
+
+        project
+        |> RemoteControl.Api.complete_struct_fields(env.document, env.position, aliases)
```

Comment:
any reason why you pass the aliases into this function rather than just calculating them where they're used? You'll safe an argument